package.json

```
@@ -10,18 +10,21 @@
     "eject": "expo eject"
   },
   "dependencies": {
-    "expo": "^30.0.1",
+    "babel-plugin-transform-react-jsx-source": "^6.22.0",
+    "expo": "^33.0.0",
     "firebase": "^5.5.4",
     "react": "16.3.1",
```

```suggestion
    "react": "16.6.0",
```
My build didn't succeed until `react` was updated as well, I had been getting an error about not being able to resolve module `scheduler/tracing`